src/AWSMetadata.jl

```diff
@@ -99,8 +99,8 @@ function _generate_high_level_wrapper(services::AbstractArray{<:OrderedDict})
         open(service_path, "w") do f
             println(f, "# This file is auto-generated by AWSMetadata.jl")
             println(f, "using AWS")
+            println(f, "using AWS: Compat")
```

Comment:
You probably want to have all of the Compat exports included here:
```suggestion
            println(f, "using AWS.Compat")
```